File: fixtures/nodejs/j13/fixture.js
// Existing codebase for feature-from-issue task
// User, AuthService, and EmailService stubs

class User {
  constructor({ id, email, name, lastPasswordChange, passwordNotificationSentAt }) {
    this.id = id;
    this.email = email;
    this.name = name;
    this.lastPasswordChange = lastPasswordChange ?? new Date();
    this.passwordNotificationSentAt = passwordNotificationSentAt ?? null;
  }

  daysSincePasswordChange() {
    const now = new Date();
    const diff = now - this.lastPasswordChange;
    return Math.floor(diff / (1000 * 60 * 60 * 24));
  }
}

class AuthService {
  constructor(userRepository) {
    this.userRepository = userRepository;
  }

  async getAllUsers() {
    return this.userRepository.findAll();
  }

  async updateUser(userId, data) {
    return this.userRepository.update(userId, data);
  }
}

class EmailService {
  async sendPasswordExpiryNotification(user) {
    // Stub: in production this would send via SMTP/SES
    console.log(`[Email] Sending password expiry notification to ${user.email}`);
    return { sent: true, to: user.email };
  }
}

module.exports = { User, AuthService, EmailService };

File: fixtures/dotnet/j13/Fixture.cs
// Existing codebase for feature-from-issue task
// User, AuthService, and EmailService stubs

public class User
{
    public string Id { get; set; } = "";
    public string Email { get; set; } = "";
    public string Name { get; set; } = "";
    public DateTime LastPasswordChange { get; set; } = DateTime.UtcNow;
    public DateTime? PasswordNotificationSentAt { get; set; } = null;

    public int DaysSincePasswordChange()
    {
        var diff = DateTime.UtcNow - LastPasswordChange;
        return (int)Math.Floor(diff.TotalDays);
    }
}

public interface IUserRepository
{
    Task<List<User>> FindAll();
    Task<User?> Update(string userId, UserUpdateData data);
}

public class UserUpdateData
{
    public DateTime? PasswordNotificationSentAt { get; set; }
}

public class AuthService
{
    private readonly IUserRepository _userRepository;

    public AuthService(IUserRepository userRepository)
    {
        _userRepository = userRepository;
    }

    public async Task<List<User>> GetAllUsers()
    {
        return await _userRepository.FindAll();
    }

    public async Task<User?> UpdateUser(string userId, UserUpdateData data)
    {
        return await _userRepository.Update(userId, data);
    }
}

public class EmailService
{
    public virtual async Task<EmailResult> SendPasswordExpiryNotification(User user)
    {
        // Stub: in production this would send via SMTP/SES
        Console.WriteLine($"[Email] Sending password expiry notification to {user.Email}");
        await Task.CompletedTask;
        return new EmailResult { Sent = true, To = user.Email };
    }
}

public class EmailResult
{
    public bool Sent { get; set; }
    public string To { get; set; } = "";
}
